api/src/pcapi/core/offers/repository.py

```diff
@@ -596,7 +596,7 @@ def _filter_collective_offers_by_statuses(query: BaseQuery, statuses: list[str]
         on_booking_status_filter.append(
             and_(
                 offer_id_with_booking_status_subquery.c.status.in_(allowed_expired_booking_status),
-                educational_models.CollectiveOffer.status == offer_mixin.CollectiveOfferStatus.EXPIRED.name,
+                educational_models.CollectiveOffer.hasEndDatetimePassed == True,
```

Comment:
Est ce qu'il ne manque pas des conditions en passant du test sur status à hasEndDatetimePassed seulement ? Le status égal à EXPIRED t'assurais par exemple que le champ validation n'avait pas certaines valeurs

Reply:
Vu ensemble -> par construction it should be fine 🤞